src/components/calcite-filter/calcite-filter.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, Element, Event, EventEmitter, Host, Prop, h } from "@stencil/core";
```

Comment:
Can you apply the change  from the master branch? Seems like filter stuff is here.